sale_elaboration/models/product_template.py

```diff
@@ -33,7 +40,7 @@ def _inverse_elaboration_profile_id(self):
 
     @api.model_create_multi
     def create(self, vals_list):
-        templates = super(ProductTemplate, self).create(vals_list)
+        templates = super().create(vals_list)
```

Comment:
Although it's correct. Let's minimize the diff to the feature changes
```suggestion
        templates = super(ProductTemplate, self).create(vals_list)
```

Reply:
All atended